CHANGELOG.md

```diff
@@ -2,748 +2,769 @@
 
 [link](https://github.com/prettier/prettier/compare/1.7.3...1.7.4)
 
-* Force template literals to break after \` for styled-components (#2926 by duailibe)
```

Comment:
@ikatyang Should we pick `*` as unordered list instead?

Reply:
I think we should do some research for this too, just like [the decision we made for strong/emphasis](https://github.com/prettier/prettier/pull/2943#issuecomment-333366584).

Reply:
Good idea. Done! https://github.com/prettier/prettier/pull/2943#issuecomment-333366584

Reply:
Opened #3025